include/spdlog/sinks/daily_file_sink.h

```diff
@@ -73,19 +72,7 @@ struct daily_filename_format_calculator
             buf.resize(buf.size() * 2);
```

Comment:
How about adding a check that the resulting buf is not too big ?
Not sure if this can actually happen, but in order to deal with situations when `strftime` keeps returning zero due to some error in formatting (or any other reason):
```c++
if(buf.size() > 4096)) 
{
  throw spdlog_ex("daily_file_sink: calc_filename() insufficient buffer space")
}

Reply:
`calc_filename()` accepts a full path name, including the path afair.